test/units/testsuite-65.sh

```diff
@@ -139,6 +139,14 @@ systemd-analyze cat-config systemd/system.conf systemd/journald.conf >/dev/null
 systemd-analyze cat-config systemd/system.conf foo/bar systemd/journald.conf >/dev/null
 systemd-analyze cat-config foo/bar
 
+# check that systemd-analyze cat-config paths work in a chroot
+mkdir -p /tmp/root
+mount --bind / /tmp/root
+systemd-analyze cat-config systemd/system-preset >/tmp/out1
+chroot /tmp/root systemd-analyze cat-config systemd/system-preset >/tmp/out2
+diff /tmp/out{1,2}
```

Comment:
I don't think we can make ASan work with the chroot stuff (at least in some reasonable way), so could you please just wrap this block in:

```suggestion
if [[ -z "${ASAN_OPTIONS:-}" ]]; then
    # check that systemd-analyze cat-config paths work in a chroot
    mkdir -p /tmp/root
    mount --bind / /tmp/root
    systemd-analyze cat-config systemd/system-preset >/tmp/out1
    chroot /tmp/root systemd-analyze cat-config systemd/system-preset >/tmp/out2
    diff /tmp/out{1,2}
fi
```

to skip the test when running with sanitizers?

Reply:
Thanks.